.codeclimate.yml

```diff
@@ -1,8 +1,8 @@
 version: 2
 plugins:
   phpcodesniffer:
-    enabled: true
-    channel: beta
```

Comment:
We should revert these changes since it doesn't look like they solved our CodeClimate woes.

Reply:
Agree